webapp-django/crashstats/settings/base.py

```diff
@@ -629,3 +629,24 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
     'SYMBOLS_BUCKET_DEFAULT_LOCATION',
     None
 )
+
+# Config for when the models pull directly from socorro.external classes.
+# NOTE: This is overwritten, for tests in settings_test.py
+SOCORRO_IMPLMENTATIONS_CONFIG = {
```

Comment:
Also, why don't you use ``DATASERVICE_CONFIG_BASE`` that Selena added in that same config file a while back? I don't really understand what in here is using her previous work, if anything at all (but I guess it would have been a lot larger if you were not building on top of her work).